openedx/features/smart_referral/tests/test_api_views.py

```diff
@@ -94,15 +99,15 @@ def test_send_initial_emails_and_save_record_refer_particular_user_once(self, in
     @file_data('data/test_data_invites.json')
     @mock.patch('webpack_loader.loader.WebpackLoader.get_bundle')
     @mock.patch('openedx.features.smart_referral.api_views.task_send_referral_and_toolkit_emails.delay')
-    def test_send_initial_emails_and_save_record_login_required(self, invites_data, mock_task_send_emails, mock_bundle):
+    def test_send_initial_emails_and_save_record_login_required(self, invites_data, mock_task_send_emails, mock_bundle):  # pylint: disable=unused-argument
```

Comment:
Instead of disabling warning, you can also do this

```
@file_data('data/test_data_invites.json')
@mock.patch('webpack_loader.loader.WebpackLoader.get_bundle', mock.Mock())
@mock.patch('openedx.features.smart_referral.api_views.task_send_referral_and_toolkit_emails.delay')
def test_send_initial_emails_and_save_record_login_required(self, invites_data, mock_task_send_emails):
```